jekyll/_cci2/examples-and-guides-overview.adoc

```diff
@@ -49,5 +49,5 @@ Use the tutorial associated with your platform to learn about the customization
 
 [#next-steps]
 == Next steps
-
 - Read the <<sample-config#,Sample config.yml Files>> document for examples of using concurrent workflows, sequential workflows, fan-in/fan-out workflows, and building Linux and iOS in one configuration file.
+- Refer to the reference guide https://circleci.com/docs/configuration-reference/ for detailed syntax.
```

Comment:
```suggestion
- Refer to the xref:configuration-reference#[Configuration reference page] for detailed syntax.
```

Reply:
@diego-assetowl thanks so much for your suggestion! Just a formatting change then we can get this added 🙏

Reply:
Great, absolutely, thanks !